platform/30-kyverno-setup.sh

```diff
@@ -19,6 +35,20 @@ kubectl create secret generic regcred --type=kubernetes.io/dockerconfigjson --fr
 kubectl create secret generic regcred --type=kubernetes.io/dockerconfigjson --from-file=.dockerconfigjson=$DOCKER_CONFIG_JSON --dry-run=client -o yaml | kubectl apply -f -
 
 # Assumes helm already installed by previous scripts
-helm repo add kyverno https://kyverno.github.io/kyverno/
-helm repo update
-helm upgrade --install kyverno kyverno/kyverno -n kyverno --create-namespace --set extraArgs="{--webhooktimeout=15,--imagePullSecrets=regcred}"
+kubectl apply -f $KYVERNO_DIR/release.yaml
+# Wait for kyverno deployment to complete
+wait_for_pods kyverno kyverno
+
+kubectl patch deployment \
+  -n kyverno \
+  kyverno \
+  --type='json' \
+  -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--webhooktimeout=15"}]'
+
+kubectl patch deployment \
+  -n kyverno \
+  kyverno \
+  --type='json' \
+  -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--imagePullSecrets=regcred"}]'
+
+  kubectl apply -f $KYVERNO_DIR/admission-control-policy
```

Comment:
```suggestion
kubectl apply -f $KYVERNO_DIR/admission-control-policy.yaml
```

Reply:
@bradbeck there are multiple files below that folder that kubectl need to apply. So there is no `.yaml`

Reply:
Ah.